addons/account/tests/test_ir_actions_report.py

```diff
@@ -1,14 +1,15 @@
 # -*- coding: utf-8 -*-
 import base64
 import io
+from PyPDF2 import PdfFileReader, PdfFileWriter
 
-from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo import Command
 from odoo.exceptions import UserError
-from odoo.tools import pdf
 from odoo.tests import tagged
-from odoo.tools import file_open
+from odoo.tools import file_open, pdf
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
```

Comment:
Why would this be imported later?
Would make sense if it was `from ../common`, but not like this

Please don't change the imports without good reason (or if you already have a lot of diff here), it will only create conflicts

Note that to follow guidelines, the import from `PyPDF2` would need to be separated from the imports of `base64` and `io`

https://peps.python.org/pep-0008/#imports

> Imports should be grouped in the following order:
> 
> 1. Standard library imports.
> 1. Related third party imports.
> 1. Local application/library specific imports.
>
> You should put a blank line between each group of imports.

Reply:
> Why would this be imported later?
> Would make sense if it was from ../common, but not like this

I think this is considered a module import? From https://www.odoo.com/documentation/17.0/contributing/development/coding_guidelines.html#imports

> The imports are ordered as
>  1. External libraries (one per line sorted and split in python stdlib)
>   2. Imports of odoo
>   3. Imports from Odoo modules (rarely, and only if necessary)

But I've checked other test files and it is not separated. So it's better to be consistent. 

I've modified the imports to be as similar to original imports as possible.